### DISCUSSION TITLE
How to use keyboardShortcuts tag?

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Trilium#2520] [Q&A]
Didn't find it in wiki, I tried

#keyboardShortcuts="Ctrl+Shift+I"
#keyboardShortcuts="ctrl i"
#keyboardShortcuts="Ctrl+I"
#keyboardShortcuts="Ctrl I"

does not work

**Comment by zadam** [comment]
It's `#keyboardShortcut="Ctrl+I"` - note the singular.

Also the frontend needs to be reloaded/restarted for changes to take effect.

**Reply by Nriver** [reply]
No wonder! Thanks, it works now.
